### PR TITLE
Fix test schema docker-compose exec exit problem 

### PR DESCRIPTION
**Pull request by metanerd**
#### Summary
 It seems like docker-compose exec behaves differently regarding exit codes compared to docker exec. I guess, I could figure it out at some stage, but instead I can just switch to docker exec instead and it behaves like we expect it to. If approved, I will remove the showcase gitlab ci modification. 

#### Release Note

```release-note

```

### CODE REVIEW

**Review by streamer45 — APPROVED** [review]
Brilliant :100:

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -129,6 +129,7 @@ test-schema-mysql:
     - k8s-memory
   rules:
     - if: '$CI_COMMIT_REF_NAME == "master"'
+    - if: '$CI_COMMIT_REF_NAME == "fix-test-schema-exit-1"'
```
Comment (metanerd) [inline .gitlab-ci.yml:132]:
```suggestion
```

File: .gitlab-ci.yml [hunk 2]
```diff
@@ -156,5 +157,6 @@ test-schema-postgres:
...
```
Comment (metanerd) [inline .gitlab-ci.yml:159]:
```suggestion
```

**Pull request merged**